supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -612,6 +612,48 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[sos-report]]
+==== image:images/caution.png[with caution] sos report (noun, adjective)
+*Description*: In RHEL 8 and later, an `_sos report_` is a collection of files that contain configuration details, system information, and diagnostic data.
+Red{nbsp}Hat Support engineers review an `sos report` when troubleshooting system problems.
+
+Write as shown: two words, monospace font.
+Optionally, append a noun such as _archive_ or _file_, especially when referring to multiple `_sos report_` files.
+(In RHEL 7 and earlier, use the one-word term `_sosreport_` instead of `_sos report_`.)
+
+To generate an `sos report` archive, run the `sos report` command (that is, the `sos` command with the `report` argument).
```

Comment:
Is this an example? An example heading or some sort of flag should be provided.

Consider swapping out parentheses for a semicolon as the semicolon provides a better contrast with the transitional phrase that follows.

> To generate an `sos report` archive, run the `sos report` command; that is, the `sos` command with the `report` argument.

Reply:
What about generating an `sos report` file? Is this a different entity in comparison to an archive. I think an _example_ heading would clear things here.

Reply:
Woth mentioning that on a Linux system one can run the command `man sos` for more info about the command and its options?

Reply:
1. It's not an example (although it does show the usage!). Moved the sentence to the initial paragraph and removed the parentheses as suggested.
2. Removed all references to _archive_ and _file_ to avoid confusion.
3. I don't think it's necessary to add this info. The point of the entries is to indicate how to write the terms.

Reply:
@bergerhoffer Do you agree?

Reply:
I agree that it's not necessary to add the man command.